### DISCUSSION TITLE
FTPおかしい

### DISCUSSION THREAD

**Discussion by kanekanegongon** [iterate-ch/cyberduck#14864] [Q&A]
今朝アップデートしたら、450エラーが頻発。

なんとかしてください。

**Comment by dkocher** [comment]
> After updating this morning, 450 errors occur frequently.

**Comment by tatze96** [comment]
Please add the following information:
* A screenshot of the error message.
* The used operating system and version of Cyberduck.
* The log file: [Enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), recreate the mentioned error, and send us the log file afterward. 

If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-14864 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file and screenshot here, make sure to attach them through the GitHub web interface, as they wouldn't be visible to us otherwise.
